tests/integration/integration_test.go

```diff
@@ -205,18 +212,26 @@ func startAnvilTestContainer() testcontainers.Container {
 
 	ctx := context.Background()
 	req := testcontainers.ContainerRequest{
-		Image: "ghcr.io/foundry-rs/foundry:nightly-3abac322efdb69e27b6fe8748b72754ae878f64d@sha256:871b66957335636a02c6c324c969db9adb1d6d64f148753c4a986cf32a40dc3c",
+		Image: "ghcr.io/foundry-rs/foundry:latest",
 		Mounts: testcontainers.ContainerMounts{
 			testcontainers.ContainerMount{
 				Source: testcontainers.GenericBindMountSource{
-					HostPath: filepath.Join(integrationDir, "../anvil/avs-and-eigenlayer-deployed-anvil-state.json"),
+					HostPath: filepath.Join(integrationDir, "../anvil/avs-and-eigenlayer-deployed-anvil-state/state.json"),
 				},
-				Target: "/root/.anvil/state.json",
+				Target: "/state.json",
 			},
 		},
 		Entrypoint:   []string{"anvil"},
-		Cmd:          []string{"--host", "0.0.0.0", "--load-state", "/root/.anvil/state.json"},
+		Cmd:          []string{"--host", "0.0.0.0", "--load-state", "/state.json"},
 		ExposedPorts: []string{"8545/tcp"},
+		HostConfigModifier: func(hostConfig *container.HostConfig) {
+    		hostConfig.PortBindings = map[nat.Port][]nat.PortBinding{
+        		"8545/tcp": {
+            	{HostIP: "0.0.0.0", HostPort: "8545"},
+        		},
+    		}
+		},
```

Comment:
What does this do?

Reply:
was debugging something. forgot to remove